python/ray/dashboard/node_stats.py

```diff
@@ -55,23 +56,19 @@ def __init__(self, redis_address, redis_password=None):
 
         super().__init__()
 
-    def _calculate_log_counts(self):
-        return {
-            ip: {
-                pid: len(logs_for_pid)
-                for pid, logs_for_pid in logs_for_ip.items()
-            }
-            for ip, logs_for_ip in self._logs.items()
-        }
+    def _insert_log_counts(self):
+        for ip, logs_by_pid in self._logs.items():
+            hostname = self._ip_to_hostname[ip]
+            logs_by_pid = {pid: len(logs) for pid, logs in logs_by_pid.items()}
+            self._node_stats[hostname]["log_count"] = logs_by_pid
 
-    def _calculate_error_counts(self):
-        return {
-            ip: {
-                pid: len(errors_for_pid)
-                for pid, errors_for_pid in errors_for_ip.items()
-            }
-            for ip, errors_for_ip in self._errors.items()
-        }
+
+    def _insert_error_counts(self):
+        for ip, errs_by_pid in self._errors.items():
+            hostname = self._ip_to_hostname[ip]
+            errs_by_pid = {pid: len(errs) for pid, errs in errs_by_pid.items()}
+            self._node_stats[hostname]["error_count"] = errs_by_pid
+            logger.info("Nodestats1: {}".format(self._node_stats[hostname]["error_count"]))
```

Comment:
What does 1 stand for?

Reply:
I didn't mean to leave this in, it was for debugging, so thank you.